### ISSUE TITLE
Data attribute?

### ISSUE THREAD

**Issue by bevels** [TheBolliwood/circleChart#1]
This is a really nice looking chart, great job. I just can't seem to figure out whether it's possible to feed it some data, or is it just cosmetic?

I understand it's possible to set the deg and startAngle which will draw a line from point A to point B, although I'm not sure what the point of that would be? Is it possible to assign a percentage based value?

**Comment by TheBolliwood** [comment]
Oh, thanks for that suggestion, I will add this feature later.
You can only give the value in degrees or radians, but you could convert the percentages until then for yourself as a hotfix. However when you set redraw to false, you can feed new data to an existing chart, if you didn't knew that.

```
function updateChart(percentages) {
    $(".circleChart").circleChart({
        deg: percentages*3.6, //percentages*360/100
        redraw: false
    });
}
```

**Comment by TheBolliwood** [comment]
Implemented in v1.0 with 5cf12cf433d6c64bc25cabf2beaa5516359bd5e8
